ci/ci/build.py

```diff
@@ -841,14 +841,14 @@ def config(self, deploy):  # pylint: disable=unused-argument
 echo done.
 '''
 
-        self.job = await batch.create_job(CI_UTILS_IMAGE,
-                                          command=['bash', '-c', script],
-                                          attributes={'name': self.name},
-                                          # FIXME configuration
-                                          service_account_name='ci-agent',
-                                          parents=self.deps_parents())
+        self.job = batch.create_job(CI_UTILS_IMAGE,
+                                    command=['bash', '-c', script],
+                                    attributes={'name': self.name},
+                                    # FIXME configuration
```

Comment:
Could we make an issue and remove fixme?

Reply:
I have no idea what the FIXME is for. @cseed?